### PR TITLE
Add basic notebook for gptassistant

### PR DESCRIPTION
**Pull request by gagb**
## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

**Review by IANTHEREAL — APPROVED** [review]
LGTM

**Review by LeoLjl — APPROVED** [review]
LGTM.

File: notebook/agentchat_oai_assistant_twoagents_basic.ipynb [hunk 1]
```diff
@@ -0,0 +1,214 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## OpenAI Assistants in AutoGen\n",
+    "\n",
+    "This notebook shows a very basic example of the `GPTAssistantAgent` working with\n",
```
Comment (qingyun-wu) [inline notebook/agentchat_oai_assistant_twoagents_basic.ipynb:9]:
```suggestion
    "This notebook shows a very basic example of the [`GPTAssistantAgent`](https://github.com/microsoft/autogen/blob/main/autogen/agentchat/contrib/gpt_assistant_agent.py#L16C43-L16C43), which is an experimental AutoGen agent class that leverages the [OpenAI Assistant API](https://platform.openai.com/docs/assistants/overview) for conversational capabilities,  working with\n",
```

Comment (qingyun-wu) [reply notebook/agentchat_oai_assistant_twoagents_basic.ipynb:9]: Mention the fact that GPTAssistantAgent is an AutoGen agent leveraging OpenAI's latest Assistant API. And add related references.

**Comment by sonichi** [comment]
Add a link in our website. Can be done in a separate PR.

**Review by qingyun-wu — APPROVED** [review]

**Pull request merged**